src/Program.cs

```diff
@@ -53,90 +76,126 @@ static int GetDimension(string dimensionType)
-                    beta = 0.320;                    
-                    break;               
-                case double x when (x > 2 && x <= 2.5):                    
-                    beta = 0.350;                    
-                    break;               
-                case double x when (x > 2.5):                    
-                    beta = 0.370;                    
-                    break;               
-            }
+            if (ratio < 0.7)
+                beta = 0.085;
+            if (ratio < 1.0)
+                beta = 0.116;
+            if (ratio < 1.5)
+                beta = 0.160;
+            if (ratio < 2.0)
+                beta = 0.260;
+            if (ratio < 2.5)
+                beta = 0.320;
+            if (ratio < 3.0)
+                beta = 0.350;
+            beta = 0.370;           
 
             double thickness = Math.Sqrt(beta * Math.Pow(verticalX, 3) * (Math.Pow(10, -5) / bendingStress));
 
             return thickness;
         }
 
-        static double CalculateBottomGlassThickness(int horizontalX, int horizontalY, int verticalZ, double bendingStress)
+        static double CalculateBottomGlassThickness(double horizontalX, double horizontalY, double verticalZ, double bendingStress)
         {
-            double ratio = (double)horizontalX / (double)horizontalY;
-            int waterPressure = CalculateWaterPressure(verticalZ);
-            
-            double beta = 0;            
+            double ratio = 0;
+            double beta = 0;
 
-            switch(ratio)
+            if (horizontalX > horizontalY)
             {
-                case double x when (x >= 0 && x <= 1):                    
-                    beta = 0.4530;                    
-                    break;                  
-                case double x when (x > 1 && x <= 1.5):                    
-                    beta = 0.5172;                    
-                    break;               
-                case double x when (x > 1.5 && x <= 2):                    
-                    beta = 0.5688;                    
-                    break;               
-                case double x when (x > 2 && x <= 2.5):                    
-                    beta = 0.6102;                    
-                    break;               
-                case double x when (x > 2.5):                    
-                    beta = 0.7134;                    
-                    break;                               
+                ratio = horizontalX / horizontalY;            
             }
+            else
+            {
+                ratio = horizontalY / horizontalX;
+            }                        
+
+            if (ratio < 1.5)
+                beta = 0.4530;
+            if (ratio < 2.0)
+                beta = 0.5172;
+            if (ratio < 2.5)
+                beta = 0.5688;
+            if (ratio < 3.0)
+                beta = 0.6102;
+            beta = 0.7134;
```

Comment:
Na ovaj način beta će uvek biti 0.7134. Možeš to proveriti u debageru ako izvršavaš korak po korak ili ako samo dodaš da se ispisuje vrednost beta a ti testiraš sa raznim odnosima stranica.